src/store/getters/state-getters.ts

```diff
@@ -344,3 +344,13 @@ export const hasBusinessNameChanged = (state: StateIF): boolean => {
 export const hasBusinessTypeChanged = (state: StateIF): boolean => {
   return state.stateModel.tombstone.entityType !== state.stateModel.originalSnapshot[0]?.business?.legalType
 }
+
+/** Check for changes between current contact and original contact. */
+export const hasContactInfoChange = (state: StateIF, rootState: any): boolean => {
+  const businessContact: BusinessContactIF = state.stateModel.defineCompanyStep.businessContact
+  const originalSnapshot: any = state.stateModel.originalSnapshot[5]
```

Comment:
@severinbeauvais I could use help here! 
I'm still having troubles defining a type on a specific index of an Array interface. I tried a few things and fell short. Any suggestions are welcome :)

Reply:
You can cast with "as".

Sometimes you have to cast to "unknown" first, then cast to target IF.

Reply:
And instead of sprinkling this snapshot[x] code everywhere, this is another reason for global getters for it.

Reply:
Try `const originalSnapshot: OriginalSnapshotIF: state.stateModel.originalSnapshot[5] as unknown`.
(untested)

Reply:
I actually did try unknown at some point, at which it worked. I wasn't sure of it's purpose though, so didn't settle on it. 

i've had mixed results with using global getters within other global getters, 
I can give it another try though!

Reply:
oo i see how this is done now 😁 
Very cool, updated in next commit!

Reply:
All getters should read from `state` right?
Example: https://github.com/bcgov/business-filings-ui/blob/84a2fbe0f8d8a3b350edba0cc7934fb63ce65d87/src/store/getters.ts#L6

Reply:
Definitely, i'm just a goofball. Passing the state on through to the global getter works great!
Also sorted out the typing :)
Check back in next commit!